site/en/docs/extensions/mv3/permission_warnings/index.md

```diff
@@ -349,17 +337,10 @@ the steps in [Viewing Warnings][27].
 
 ## Update permissions {: #update_permissions }
 
-Updating an extension with additional permissions may temporarily disable it. The user will have to
+When an extension is updated to include a new  permission that [triggers a warning][section-warnings] it may temporarily disable it. The user will have to
```

Comment:
```suggestion
When an extension is updated to include a new permission that [triggers a warning][section-warnings] it may temporarily disable it. The user will have to
```